content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -87,9 +87,8 @@ This output matches the span created in the preceding code sample.
 
 ## ASP.NET Core
 
-The following sample demonstrates automatic and manual
-[tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry) with ASP.NET
-Core.
+The following sample demonstrates [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry)
```

Comment:
Minor - use example instead of sample to avoid potential confusion with "sampling/sampler":

```suggestion
The following example demonstrates [tracing](/docs/concepts/signals/traces/#tracing-in-opentelemetry)
```

Reply:
Addressed https://github.com/open-telemetry/opentelemetry.io/pull/1565/commits/af71246dfd68f4bc496d3705c73317f6d580e3cc